.github/workflows/refund-nightly-skyline.yml

```
@@ -0,0 +1,39 @@
+---
+name: Refund Nightly Local Tests
+on: # yamllint disable-line rule:truthy
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  refund_ci:
+    name: Refund CI
+    uses: ./.github/workflows/apex-ci.yml
+    with:
+      lint: true
+      unit_test: true
+      e2e_apex_test: true
+      e2e_skyline_test: false
+      e2e_refund_test: true
+      apex_bridge_ref: feat/skyline_refund
+      blade_apex_bridge_ref: feat/skyline_refund
+      e2e_apex_big_test: ${{ vars.RUN_E2E_BIG_TESTS }}
+      e2e_apex_skip_redundant_tests: ${{ vars.SKIP_E2E_REDUNDANT_TESTS }}
+    secrets:
+      PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+  notification_refund_nightly:
+    name: Refund Nightly Notification
+    uses: ./.github/workflows/refund-notification-nightly.yml
```

```suggestion
    uses: ./.github/workflows/refund-notification-nightly-skyline.yml
```
we need to create a new notification file similar to `refund-notification-nightly.yml` but with the `header` text **Skyline Refund Main Nightly Local Tests**, and it should display the status of `e2e_skyline_test_output` just like it currently does for `e2e_apex_test_output` and `e2e_refund_test_output`